src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -143,7 +143,7 @@ export async function getCowSwapTradeQuote(
       contractAddress: sellAssetErc20Address,
     })
 
-    const [feeDataOptions, sellAssetUsdRate] = await Promise.all([
+    const [{ average }, sellAssetUsdRate] = await Promise.all([
```

Comment:
Feels like too much magic with this destructuring but why not

Reply:
meh will pull it out just for clarity